src/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ ENDIF()
 include(kokkoskernels_eti.cmake)
 SET(ETI_HEADERS)
 
+MESSAGE(STATUS "devices: ${DEVICES}")
```

Comment:
@lucbv Is this a temporary print that should be taken out? The cmake prints out a list of devices in this list:
```
=======================
KokkosKernels ETI Types
   Devices:  <Serial,HostSpace>
   Scalars:  double;complex<double>
   Ordinals: int
   Offsets:  int;size_t
   Layouts:  LayoutRight

KokkosKernels TPLs
=======================
```